src/classes/OPP_OpportunityContactRoles_TEST.cls

```diff
@@ -912,4 +912,217 @@ private class OPP_OpportunityContactRoles_TEST {
         System.assertNotEquals(null, result[0].Opportunity.AccountId);
     }
 
+    /******************************************************************************************************
```

Comment:
These are a lot of new test methods to add to an already large test class. Could these belong in a different and related (smaller) test class, or maybe they belong in a new class entirely to group the testing logic together.